reactComponents/src/searchbar/redux/tokenSearchSlice.ts

```diff
@@ -8,6 +8,23 @@ import config from '../config';
 
 const LOAD_LIMIT = Number(config.LOAD_LIMIT || 10);
 
+// 
+const isDuplicatedExchange = (exchange, networks, networkMap) => {
+  let isDuplicated = false;
+
+  networks.forEach((network) => {
```

Comment:
I don't think it will cause any performance issues but this smells like inefficient code